bridge_test.go

```diff
@@ -51,6 +53,47 @@ func TestHandlePacket(t *testing.T) {
 				&TimerEvent{
 					metricName: "foo",
 					value:      200,
+					labels:     map[string]string{},
+				},
+			},
+		}, {
+			name: "datadog tag extension",
+			in:   "foo:100|c|#tag1:bar,tag2:baz,tag3,tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": "<n/a>", "tag4": "<n/a>"},
```

Comment:
I'm still unsure about the tag support. Are you using that in your setup? I'd assume these are rarely static names with only one possible value. Querying metrics with varying label sets (different label keys) will result in all kind of problems.

I'd suggest to either forbid the usage of tags or use them as value instead. For example:

```
# input
foo:100|c|#a:b,foo,bar
foo:300|c|#a:d,qux,bar

# metrics
foo{a="b", tag2="foo", tag3="bar"} 100
foo{a="b", tag2="qux", tag3="bar"} 300
```

This still leaves the problem that the caller has to ensure the order is stable.